### DISCUSSION TITLE
Documentation for Postman Flows

### DISCUSSION THREAD

**Discussion by Shubh0405** [postmanlabs/postman-flows#49] [Q&A] [answered]
Hi, 

I have been trying to use postman flows but can't find any online resource or documentation about how to get started with flows. Is there any documentation made where I can find what all things we can do with flows and how to use all blocks and stuff??

**Comment by samay-v** [comment, marked as answer]
Docs are in progress, and should be out shortly. In the meantime, here’s part of a livestream that walks through it slowly: [Flows to control your workflow graphically](https://www.youtube.com/watch?v=j-fkJLbOpjk&t=2840s).

Also, every block has a "View information" option as well that tells you what the bock and each of the ports on the block does. 

![image](https://user-images.githubusercontent.com/28357106/138207274-3b334286-021a-4d8b-b02e-06b6245d0b3e.png)
![image](https://user-images.githubusercontent.com/28357106/138207302-0a421ac8-5bce-4db3-896b-4e5b6fe6c2bf.png)

**Comment by Shubh0405** [comment]
Cool, I will look into the live stream. Thanks for the help :100:
